app/controllers/decidim/participatory_documents/admin/documents_controller.rb

```diff
@@ -74,6 +81,10 @@ def add_snippets
         def document
           @document ||= Decidim::ParticipatoryDocuments::Document.find_by(component: current_component)
         end
+
+        def documents
+          @documents ||= Decidim::ParticipatoryDocuments::Document.where(component: current_component)
+        end
```

Comment:
```suggestion
        def annotations
          @annotations ||= Decidim::ParticipatoryDocuments::Annotations.where(document: document)
        end
```

We won't be listing documents here (there will be only one document per component). Instead we are going to use this for listing the participation groups. As groups are being created in the #15, we can put annotations directly or just a placeholder.